design-layout/css-layout-position-assignmnet-3/css-layout.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="cssLayout.css">
+    <title>Css layout assignmnet-3</title>
+</head>
+<body>
+    <div class="header">
+        <div class="local-content">
+            <p class="bold">Local Plumber</p>
+            <p class="italic">Trustworthy and insured</p>
+            <p class="contact">1-800-555-1800</p>
+        </div>
+        <div class="free-trial-form">  
+            <form>
```

Comment:
Try to use other way to write the form we dont need table inside a form .

Reply:
next time i will used div tag